sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PortEvaluationStage.java

```diff
@@ -232,7 +235,10 @@ protected void setProtos(PodInfoBuilder podInfoBuilder, Protos.Resource resource
   }
 
   private static Optional<Integer> selectDynamicPort(
-      MesosResourcePool mesosResourcePool, PodInfoBuilder podInfoBuilder, String preReservedRole)
+      MesosResourcePool mesosResourcePool,
+      PodInfoBuilder podInfoBuilder,
+      String preReservedRole,
+      PortSpec spec)
   {
```

Comment:
based on previous PR reviews I understand that dcos-commons has it's own unique style of bracket alignment... I'll state again how much I dislike it.   It should all be right aligned or match... not both.   Too many cases to remember.   To be consistent with other orchestration projects it is all right aligned.   not a blocker.

Reply:
ya this is something enforced by our linter, open to change? (maybe when we move to the USI)